src/pkg/cluster/injector.go

```diff
@@ -123,11 +127,15 @@ func (c *Cluster) StartInjection(ctx context.Context, tmpDir, imagesDir string,
 	}
 
 	spinner.Success()
+	l.Debug("done with injection", "duration", time.Since(start))
```

Comment:
🤔 Not in scope for this PR even remotely, but I wonder how difficult it would be to start aggregating duration metrics from CI logs. We don't get a ton of runtime feedback cause _airgap_ so feel like we need to get the signal that we can.

Reply:
Yeah that'd be interesting, I believe we save log files somewhere or if not it's not hard to do with actions